File: C programming/Assignment/Assignment4/primerange.c
#include <stdio.h>
 void main(){
    int n,start,flag;
    printf("enter the number:");
    scanf("%d",&n);

    for(int start = 2;start<=n;start++){
        int flag = 0;
        for(int i =2;i<start;i++){
            if(start%i==0){
                flag=1;
                break;
            }
        }    
        if(flag==0){
            printf("%d ",start);
        }
    }
 } 

File: Java_programming/basic_java/assignment4/primerange.java
package ssignment4;

public class primerange {

	public static void main(String[] args) {
		int n=100,start,flag;
//	    printf("enter the number:");
//	    scanf("%d",&n);

	    for( start = 2;start<=n;start++){
	        flag = 0;
	        for(int i =2;i<start;i++){
	            if(start%i==0){
	                flag=1;
	                break;
	            }
	        }    
	        if(flag==0){
	            System.out.printf("%d ",start);
	        }
	    }

	}

}
